mkt/search/filters.py

```diff
@@ -204,6 +205,23 @@ class ExtensionSearchFormFilter(SearchFormFilter):
     VALID_FILTERS = ['author.raw']
 
 
+class HomescreenFilter(BaseFilterBackend):
+    """
+    A django-rest-framework filter backend that filters in or out homescreens.
+    """
+    def filter_queryset(self, request, queryset, view):
+        is_homescreen = request.GET.get('is_homescreen')
```

Comment:
maybe we should check webapp_type === 'homescreen' rather than is_homescreen === '0'